modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize and configure an existing project access roles name by editing the YAML code, and add an `console.openshift.io/display-name: Display name` annotation to the cluster.
+
+.Procedure
+To customize the cluster role options of all projects that are shown in the *Project Access* tab:
+
+. Navigate to the *Search* view.
+. In the *Search* view, select the *Resources* drop down and search *Console*.
+. Select the *Console `operator.openshift.io/v1`* to add as a new row in resource list.
```

Comment:
@rishumehra Now that you've provided the preview of a rendered doc with the screenshots, I'm afraid I got confused by this line a bit.
It looks like the step suggests adding a new row that will be "Console `operator.openshift.io/v1". Am I misreading it? Does my suggestion make sense?

```suggestion
. Select *Console `operator.openshift.io/v1`* in the Resources list (to add a new role? Is this tail needed?)
```

Reply:
@missmesss Let me demo it to you for this one, then we can discuss this further. Because now I am also not sure about it.

Reply:
@rishumehra Do you mean some video demo or presentation? We could save you some time as anyway I don't need to know how it works in detail. It'll be great if you check that this line is accurate, that'll be enough, thank you.

Reply:
@missmesss 

Here is a procedure overview for your reference:

- Actually in *Resource* tab we search for "Console" and select the "Console `operator.openshift.io/v1" resource. Like this: 
  ![image](https://user-images.githubusercontent.com/3268242/151935077-28977e81-9c67-4b44-b850-4173bbf51f0e.png)

- Then add ""Console `operator.openshift.io/v1"" item to a resource list, like this:
 ![image](https://user-images.githubusercontent.com/3268242/151935715-658198cd-8cd7-4622-bfc4-7879de58587d.png)

- Then inside it we edit the YAML code to edit the list of user access roles.
  ![image](https://user-images.githubusercontent.com/3268242/151935913-713ddccc-1704-4e64-baba-65c016a49361.png)

Reply:
To simplify I'll just keep: 

. Select the **Console `operator.openshift.io/v1`**.